workspaces/azure-devops/app-config.yaml

```diff
@@ -55,6 +55,11 @@ catalog:
 
     # Local example scaffolder action
     - type: file
-      target: ../../examples/scaffolder.yaml
+      target: ../../examples/scaffolder/scaffolder.yaml
```

Comment:
Could we call this one `dotnet.yaml`? I like how you named the other one more specifically.

Reply:
Sure, I've renamed it :)